packages/core/src/sync/sync-api.ts

```diff
@@ -186,21 +198,37 @@ export class SyncApi implements ISyncApi {
    * Also keeps in sync with new anchors.
    */
   async startModelSync(
-    startBlock: number,
-    endBlock: number,
-    models: string | string[]
+    models: StreamID | StreamID[],
```

Comment:
Do these need to be stream IDs please?
Looks like they get converted to string, so couldn't the input be a string directly?